### PR TITLE
Ramanenka_tasks

### PR DESCRIPTION
**Pull request by margoroma**

### CODE REVIEW

File: Tasks/RM_HomeTask2+.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+                h_ = str(dict_hour.get(int(h)))
+
+                if m1_1 > 0 and m1_0 in range(2,6):           # Проверка 2-х цифр в минутах
+                    m_dec = d_0.get(m1_0)
+                    m_one = dict_min.get(m1_1)
+                    m_ = f'{m_dec} {m_one}'
+                    for key in d_m.keys():                    # Вывод слова "минут"
+                        if m1_1 in key:
+                            mins = d_m.get(key)
```
Comment (MikitaTsiarentsyeu) [inline Tasks/RM_HomeTask2+.py:153]: image

There will be an error for the value 12:00

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]

**Comment by MikitaTsiarentsyeu** [comment]
And now test it for value 24:00

**Comment by margoroma** [comment]
А зараз бачна?)

**Comment by margoroma** [comment]
Last-Version

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Great job, I would use the same logic. Explicitly showing time format and validation rules are also a good decisions. Your score is 9.5.

**Pull request merged**